Mirror/Runtime/NetworkManager.cs

```diff
@@ -648,16 +648,8 @@ void OnServerAddPlayerInternal(NetworkConnection conn)
                 return;
             }
 
-            GameObject player;
             Transform startPos = GetStartPosition();
-            if (startPos != null)
-            {
-                player = Instantiate(playerPrefab, startPos.position, startPos.rotation);
-            }
-            else
-            {
-                player = Instantiate(playerPrefab, Vector3.zero, Quaternion.identity);
-            }
+            GameObject player = startPos != null ? Instantiate(playerPrefab, startPos.position, startPos.rotation) : Instantiate(playerPrefab, Vector3.zero, Quaternion.identity);
```

Comment:
not sure about this one,  line becomes very log and hard to read.

Reply:
i'm also not sure, but i don't think that it is too long

Reply:
break it up into multiple lines,  then it becomes more readable.